TTS/tts/models/xtts.py

```diff
@@ -255,39 +255,57 @@ def device(self):
         return next(self.parameters()).device
 
     @torch.inference_mode()
-    def get_gpt_cond_latents(self, audio, sr, length: int = 3):
+    def get_gpt_cond_latents(self, audio, sr, length: int = 30, chunk_length: int = 6):
```

Comment:
I think the default value here and in the config should be the same.

Reply:
why?

Reply:
I think it will be better, users can  try to call this function individually and then get very different results. I think the better is both be equal to avoid issues like it.

Reply:
config as in the code or the release model's config